zypp-plugin/forwarding_zypp_plugin.cc

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (c) 2020 SUSE LLC
+ *
+ * All Rights Reserved.
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of version 2 of the GNU General Public License as published
+ * by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+ * more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, contact SUSE LLC.
+ *
+ * To contact SUSE about this file by physical or electronic mail, you may
+ * find current contact information at www.suse.com.
+ */
+
+#include <string>
+using namespace std;
+
+#include <boost/process.hpp>
+namespace bp = boost::process;
+
+#include "zypp_plugin.h"
+
+class ForwardingZyppPlugin : public ZyppPlugin {
```

Comment:
Please explain the purpose of the class.

Reply:
Ah, you're right. See above now.